### ISSUE TITLE
Warning when compiling

### ISSUE THREAD

**Issue by mitar** [jerome-pouiller/reredirect#19]
```
attach.c: In function ‘child_open’:
attach.c:94:9: warning: ‘strncat’ specified bound 4097 equals destination size [-Wstringop-overflow=]
   94 |         strncat(buf, "/", sizeof(buf));
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by jerome-pouiller** [comment]
Fixed with a1fefa4

**Comment by mitar** [comment]
Thanks!
